core/data/src/main/java/app/aaps/core/data/iob/GlucoseStatus.kt

```diff
@@ -6,5 +6,15 @@ data class GlucoseStatus(
     val delta: Double = 0.0,
     val shortAvgDelta: Double = 0.0,
     val longAvgDelta: Double = 0.0,
-    val date: Long = 0L
+    val date: Long = 0L,
```

Comment:
please add javadoc here to explain meaning of new values

Reply:
Sure; what is javadoc?